tests/pvc_utils.go

```diff
@@ -0,0 +1,74 @@
+package tests
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	defaultTimeout = 30*time.Second
+)
+
+// Creates a standard PVC in the passed in namespace.
+func CreateDefaultPVC(client *kubernetes.Clientset, namespace string, name string, size string) *k8sv1.PersistentVolumeClaim {
+	return CreatePVCFromDefinition(client, namespace, NewPVCDefinition(name, size, nil))
+}
+
+// Creates a PVC in the passed in namespace from the passed in PersistentVolumeClaim definition.
+func CreatePVCFromDefinition(client *kubernetes.Clientset, namespace string, def *k8sv1.PersistentVolumeClaim) *k8sv1.PersistentVolumeClaim {
+	var pvc *k8sv1.PersistentVolumeClaim
+	var err error
+	if wait.PollImmediate(2*time.Second, defaultTimeout, func() (bool, error) {
+		pvc, err = client.CoreV1().PersistentVolumeClaims(namespace).Create(def)
+		if err != nil {
+			return false, nil
+		}
+		return true, nil
+	}) != nil {
+		Fail("Unable to create PVC: " + def.GetName())
```

Comment:
Would it be useful to report the error produced by `wait.PollImmediate` (presumably a timeout)?

Reply:
So my initial reaction was, no since it will always be a timeout, all the other errors are consumed inside the loop. But then my second reaction is, but if someone that doesn't know the code sees the error, they will assume something happened besides a timeout. And there is no harm in showing the error, so why not.